installer/scripts/omsadmin.sh

```diff
@@ -424,11 +449,28 @@ onboard()
 
     save_config
 
-    # If a test is not in progress then:  
-    if [ -z "$TEST_WORKSPACE_ID" -a -z "$TEST_SHARED_KEY" ]; then
-        # Register omsagent as a service and start the agent
-        /opt/microsoft/omsagent/bin/service_control start 
+    copy_omsagent_conf
+
+    if [ -z "$MULTI_HOMING_MARKER" ]; then
+        # update the default folders when onboard to a workspace as primary
+        # this is the default behavior
+        update_symlinks
+    else
+        # do not update the default folders when onboard the workspace as secondary
+        # leave a marker in the etc folder of the workspace to indicate the partner
+        echo $MULTI_HOMING_MARKER > $CONF_DIR/.multihoming_marker
+    fi
+
+    configure_syslog
 
+    configure_monitor_agent
+
+    configure_logrotate
+
+    # If a test is not in progress then register omsagent as a service and start the agent 
+    if [ -z "$TEST_WORKSPACE_ID" -a -z "$TEST_SHARED_KEY" ]; then
+        $SERVICE_CONTROL start $WORKSPACE_ID 
+    fi
```

Comment:
This closing fi needs to be removed so that omsconfig and omsConsistencyInvoker are set up when tests are not run.